models/topics.model.js

```diff
@@ -1,10 +1,17 @@
 const db = require('../db/connection.js');
+const endpoints = require('../endpoints.json')
+const fs = require('fs/promises');
 
-exports.selectTopics = () => {
+
+exports.fetchTopics = () => {
 
     let query = `SELECT * FROM topics;`
 
     return db.query(query).then((result) => {
         return result.rows;
         });
+}
+
+exports.fetchEndpoints = () => {
```

Comment:
we don't need a model for this, the functionality we have here can go in the controller